pages/linux/hashcat.md

```diff
@@ -0,0 +1,32 @@
+# hashcat
+
+> Fast and advanced password recovery tool.
+> More information: <https://hashcat.net/hashcat/>.
+
+- Brute-force attack with default hashcat mask:
+
+`hashcat --hash-type {{id_hash_type}} --attack-mode 3 {{hash}}`
+
+- Brute-force attack with a known pattern of 4 digits:
+
+`hashcat --hash-type {{id_hash_type}} --attack-mode 3 {{hash}} ?d?d?d?d`
+
+- Brute-force attack with max length of 8 chars using the complete charset (numbers + letters + special chars):
```

Comment:
```suggestion
- Brute-force attack with max length of 8 chars using the complete ASCII charset:
```
Is this still correct?

Reply:
Almost, from the help page:   
   l | abcdefghijklmnopqrstuvwxyz
  u | ABCDEFGHIJKLMNOPQRSTUVWXYZ
  d | 0123456789
  s |  !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
  a | ?l?u?d?s

this mean "?a" cover ASCII from 32 to 126

Reply:
I replaced it with "all printable ASCII chars"